webpack.config.js

```diff
@@ -1,95 +1,103 @@
-        path: path.resolve(__dirname, 'assets/js'),
-        filename: 'bundle.js'
-    },
-
-    module: {
-        rules: [
-            {
-                test: /\.js$/,
-                exclude: /node_modules/,
-                loader: 'babel-loader',
-                options: {
-                    cacheDirectory: true
-                }
-            },
-
-            // the url-loader uses DataUrls.
-            // the file-loader emits files.
-            {
-                test: /\.woff(2)?(\?v=[0-9]\.[0-9]\.[0-9])?$/,
-                loader: 'url-loader',
-                options: {
-                    limit: 10000,
-                    minetype: 'application/font-woff',
-                }
-            },
-            {
-                test: /\.(ttf|eot|svg)(\?v=[0-9]\.[0-9]\.[0-9])?$/,
-                loader: 'file-loader'
-            },
-            {
-                test: /\.css$/,
-                loader: 'style-loader!css-loader'
-            },
-            {
-                test: /\.(png|jpg)$/,
-                loader: 'url-loader',
-                options: {
-                    // inline base64 URLs for <=8k images, direct URLs for the rest
-                    limit: 8192,
-                }
-            }
-        ]
-    },
-
-    resolve: {
-        modules: [
-            path.resolve(__dirname, 'assets/js'),
-            path.resolve(root, 'assets/js'),
-            'node_modules'
-        ]
-    },
-
-    plugins: [
-        new webpack.ProvidePlugin({
-            'jQuery': 'jquery'
-        }),
-
-        new webpack.DllReferencePlugin({
-            sourceType: 'this',
-            manifest: require(path.resolve(root, 'assets/dist/carbon.vendor.json')),
-        }),
+	entry: {
+		bundle: './src/index.js'
+	},
+	output: {
+		path: path.resolve( __dirname, 'build' ),
+		filename: isProduction ? '[name].min.js' : '[name].js'
+	},
+	module: {
+		rules: [
+			{
+				test: /\.js$/,
+				exclude: /node_modules/,
+				use: {
+					loader: 'babel-loader',
+					options: {
+						cacheDirectory: true
+					}
+				}
+			},
+			{
+				test: /\.scss$/,
+				use: [
+					MiniCssExtractPlugin.loader,
+					{
+						loader: 'css-loader',
+						options: {
+							importLoaders: 2
+						}
+					},
+					{
+						loader: 'postcss-loader'
+					},
+					{
+						loader: 'sass-loader'
+					}
+				]
+			}
+		]
+	},
+	externals: [
+		'@wordpress/compose',
+		'@wordpress/data',
+		'@wordpress/element',
+		'@wordpress/hooks',
+		'@wordpress/i18n',
+		'classnames',
+		'lodash'
+	].reduce( ( memo, name ) => {
+		memo[ name ] = `cf.vendor['${ name }']`;
 
-        new webpack.DllReferencePlugin({
-            context: root,
-            sourceType: 'this',
-            manifest: require(path.resolve(root, 'assets/dist/carbon.core.json'))
-        })
-    ],
+		return memo;
+	}, {
+		'@carbon-fields/core': 'cf.core'
+	} ),
+	plugins: [
+		new MiniCssExtractPlugin( {
+			filename: isProduction ? '[name].min.css' : '[name].css'
```

Comment:
You create `.min.css` file on production, but in PHP enqueue `.css` file in function `admin_enqueue_scripts`